.travis.yml

```diff
@@ -28,6 +28,10 @@ matrix:
        env: TOXENV=py37
      - python: 3.7
        env: TOXENV=py37-functional
+     - python: 3.8-dev
```

Comment:
3.8-dev is still a development build and is not ready for production yet.

Reply:
But it's good enough for testing purposes.

Reply:
The reason we want to test a certain version is that we’ll be confident supporting this version in PyPI wheels. Therefore, I would stick to stable release only.